extensions/amp-story/1.0/amp-story.js

```diff
@@ -344,16 +350,34 @@ export class AmpStory extends AMP.BaseElement {
   }
 
 
-  /** @private */
+  /** @override */
+  pauseCallback() {
+    // Store the current paused state, to make sure the story does not play on
+    // resume if it was previously paused.
```

Comment:
Why don't we want the story to play when `resume` is called? Not sure I understand.

Reply:
We need to restore the `PAUSED_STATE`: if it was already paused, it needs to remain paused.

Example:
- User loads the story and opens the share menu, which pauses the story
- User leaves the browser to do something else on his phone
- User comes back to the story, which needs to remain paused

Reply:
Oh got you. Thanks!